Examples/SentryExampleWinUI/Application.swift

```diff
@@ -55,7 +84,8 @@ public class SentryApplication: SwiftApplication {
         panel.children.append(makeButton("fatalError()") { fatalError("Boom goes the dynamite!") })
         panel.children.append(makeButton("[0][1]") { _ = [0][1] })
         panel.children.append(makeButton("RaiseFailFastException()") { RaiseFailFastException(nil, nil, 0) })
-        panel.children.append(makeButton("Report empty EXCEPTION_POINTERS", onClick: { SentrySDK.capture(exception: EXCEPTION_POINTERS()) }))
+        panel.children.append(stowedExceptionButton)
+        panel.children.append(makeButton("Report custom exception", onClick: { _ = SentrySDK.captureException(type: "CustomException", description: "exception created from an example") }))
```

Comment:
isn't the expected usage to call this in the `unhandledException` of the app?

Reply:
Yep, I added both - the second one shows details of the exception. For some reason the captureException in the unhandledException does not print the exception to the log like the other exceptions, thus I added the extra button to construct the exception, which prints it as expected.